website/docs/r/opa_version.html.markdown

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
Please use [our preview tool](https://github.com/hashicorp/terraform-provider-tfe/blob/main/docs/changelog-process.md#updating-the-documentation) to paste your markdown and render a view of docs. Then, add a screenshot to your PR description. This helps review how the doc will render.